common.lua

```diff
@@ -183,7 +183,7 @@ function pipeworks.create_fake_player(def, is_dynamic)
 		is_player = delay(true),
 		is_fake_player = true,
 
-		_formspec = def.formspec or default.gui_survival_form,
+		_formspec = def.formspec or "size[8,8.5]label[1,1;404 formspec not found]",
```

Comment:
This could probably be just empty string.
Got any idea why it is there in first place?

Reply:
probably because it trying to be a fake player as much as possible? dunno, dont think anyone ever cared to check it, so ill just set it to `or ""`

Reply:
https://github.com/mt-mods/pipeworks/pull/16/commits/58bd68f79d3061f66b0df679c35ff6594ba56710 resolved